bundle.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
```

Comment:
I think this file should be moved to the `build/` directory

Reply:
`operator-sdk bundle create --generate-only`
command created this file by default at root. No flag to change the directory of dockerfile.